docs/content/querying/groupbyquery.md

```diff
@@ -250,6 +250,7 @@ When using the "v2" strategy, the following query context parameters apply:
 |`sortByDimsFirst`|Sort the results first by dimension values and then by timestamp.|
 |`forcePushDownLimit`|When all fields in the orderby are part of the grouping key, the broker will push limit application down to the historical nodes. When the sorting order uses fields that are not in the grouping key, applying this optimization can result in approximate results with unknown accuracy, so this optimization is disabled by default in that case. Enabling this context flag turns on limit push down for limit/orderbys that contain non-grouping key columns.|
 |`forceHashAggregation`|Force to use hash-based aggregation.|
+|`forceSingleThreadedCombine`|Force to use single-threaded combine.|
```

Comment:
Is it really needed to have this config? Do you imagine anybody will turn it on?

Reply:
@leventov I removed this in the latest patch. Now, Druid can decide to choose the combining strategy by itself. It first merges the dictionaries generated by underlying SpillingGroupers and check the size of the merged dictionary exceeds a given threshold. If it's not, parallel combining strategy is chosen.